src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  background-color: #50464634;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__item {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  margin-right: 20px;
+  text-transform: uppercase;
+}
+
+.nav__item.lastitem {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  display: flex;
+  position: absolute;
+  width: 37px;
+  height: 4px;
+  top: 56px;
```

Comment:
```suggestion
  bottom: 0;
```

Reply:
Then pseudo element dissapiars

Reply:
add the position relative for active link to avoid disappearing